2023-02-21 18:36:14,735 INFO    MainThread:12089 [wandb_setup.py:_flush():68] Configure stats pid to 12089
2023-02-21 18:36:14,735 INFO    MainThread:12089 [wandb_setup.py:_flush():68] Loading settings from /home/andrea/.config/wandb/settings
2023-02-21 18:36:14,735 INFO    MainThread:12089 [wandb_setup.py:_flush():68] Loading settings from /home/andrea/PycharmProjects/data_exfiltration_tabular/wandb/settings
2023-02-21 18:36:14,735 INFO    MainThread:12089 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-02-21 18:36:14,736 INFO    MainThread:12089 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': '/home/andrea/PycharmProjects/data_exfiltration_tabular/train.py'}
2023-02-21 18:36:14,736 INFO    MainThread:12089 [wandb_init.py:_log_setup():492] Logging user logs to /home/andrea/PycharmProjects/data_exfiltration_tabular/wandb/run-20230221_183614-gzc3eqm6/logs/debug.log
2023-02-21 18:36:14,736 INFO    MainThread:12089 [wandb_init.py:_log_setup():493] Logging internal logs to /home/andrea/PycharmProjects/data_exfiltration_tabular/wandb/run-20230221_183614-gzc3eqm6/logs/debug-internal.log
2023-02-21 18:36:14,737 INFO    MainThread:12089 [wandb_init.py:init():532] calling init triggers
2023-02-21 18:36:14,737 INFO    MainThread:12089 [wandb_init.py:init():539] wandb.init called with sweep_config: {}
config: {}
2023-02-21 18:36:14,738 INFO    MainThread:12089 [wandb_init.py:init():588] starting backend
2023-02-21 18:36:14,738 INFO    MainThread:12089 [wandb_init.py:init():592] setting up manager
2023-02-21 18:36:14,753 INFO    MainThread:12089 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-02-21 18:36:14,760 INFO    MainThread:12089 [wandb_init.py:init():599] backend started and connected
2023-02-21 18:36:14,766 INFO    MainThread:12089 [wandb_init.py:init():687] updated telemetry
2023-02-21 18:36:14,783 INFO    MainThread:12089 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-02-21 18:36:15,556 INFO    MainThread:12089 [wandb_run.py:_on_init():2134] communicating current version
2023-02-21 18:36:15,690 INFO    MainThread:12089 [wandb_run.py:_on_init():2143] got version response 
2023-02-21 18:36:15,690 INFO    MainThread:12089 [wandb_init.py:init():775] starting run threads in backend
2023-02-21 18:36:15,798 INFO    MainThread:12089 [wandb_run.py:_console_start():2114] atexit reg
2023-02-21 18:36:15,800 INFO    MainThread:12089 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-02-21 18:36:15,801 INFO    MainThread:12089 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-02-21 18:36:15,802 INFO    MainThread:12089 [wandb_run.py:_redirect():2059] Redirects installed.
2023-02-21 18:36:15,803 INFO    MainThread:12089 [wandb_init.py:init():817] run started, returning control to user process
2023-02-21 18:36:35,674 INFO    MainThread:12089 [wandb_run.py:_finish():1854] finishing run siposova-andrea/Data Exfiltration Attacks and Defenses/gzc3eqm6
2023-02-21 18:36:35,674 INFO    MainThread:12089 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-02-21 18:36:35,674 INFO    MainThread:12089 [wandb_run.py:_restore():2066] restore
2023-02-21 18:36:35,675 INFO    MainThread:12089 [wandb_run.py:_restore():2072] restore done
2023-02-21 18:36:35,677 INFO    MainThread:12089 [wandb_run.py:_restore():2066] restore
2023-02-21 18:36:35,677 INFO    MainThread:12089 [wandb_run.py:_restore():2072] restore done
2023-02-21 18:36:36,682 ERROR   MainThread:12089 [wandb_run.py:_atexit_cleanup():2104] Problem finishing run
Traceback (most recent call last):
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 2092, in _atexit_cleanup
    self._on_finish()
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 2323, in _on_finish
    self._exit_code, settings=self._settings, printer=self._printer
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 3239, in _footer_exit_status_info
    printer.display(f'{" ".join(info)}')
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/lib/printer.py", line 68, in display
    self._display(text, level=level, default_text=default_text)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/lib/printer.py", line 152, in _display
    self._display_fn_mapping(level)(text)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/errors/term.py", line 44, in termlog
    silent=not _show_info,
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/errors/term.py", line 95, in _log
    click.echo(line, file=sys.stderr, nl=newline)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/click/utils.py", line 298, in echo
    file.write(out)  # type: ignore
BrokenPipeError: [Errno 32] Broken pipe
